packages/kbn-timerange/kibana.jsonc

```diff
@@ -1,9 +1,7 @@
 {
   "type": "shared-common",
   "id": "@kbn/timerange",
-  "owner": [
-    "@elastic/obs-ux-logs-team"
-  ],
-  "group": "observability",
-  "visibility": "private"
-}
\ No newline at end of file
+  "owner": ["@elastic/obs-ux-logs-team"],
+  "group": "platform",
```

Comment:
I couldn't use the package outside of observability so I updated the group in order to be reuse and extend it. Let me know if that's okay for you @elastic/obs-ux-logs-team

Reply:
That was the plan in case the package is needed elsewhere so it's fine.